internal/cache/metadata/stat_cache.go

```diff
@@ -84,11 +82,11 @@ type entry struct {
 }
 
 func (e entry) Size() uint64 {
-	return uint64(unsafe.Sizeof(gcs.Object{}) + unsafe.Sizeof(entry{}))
+	return 1
```

Comment:
can we please search and remove changes like https://github.com/GoogleCloudPlatform/gcsfuse/blob/read_cache_release/internal/gcsx/bucket_manager.go#L88 also ?

Reply:
done